Userland/Applets/Audio/main.cpp

```diff
@@ -83,8 +83,18 @@ class AudioWidget final : public GUI::Widget {
                 window()->resize(44, 16);
                 m_percent_box->set_tooltip("Hide percent");
             }
-            reposition_slider_window();
+            // FIXME: We want to move the slider widget to be above the loudspeaker symbol of the
+            // taskbar widget, but that moves when the percentage display is toggled. For this
+            // reason, we need to read back the new screen position of the taskbar widget, i.e.
+            // window(). However, due to a bug in WindowServer whose origin is not entirely known,
+            // the reported screen position of the window is incorrect for up to one frame randomly,
+            // probably also involving a race condition. Taking the window rect at face value will
+            // lead to incorrect positioning of the slider window. To circumvent this issue
+            // temporarily, we update the position twice, because the reported screen position is
+            // always correct two frames later. Therefore, this variable will lead to two position updates.
```

Comment:
Please don't do this kind of hack. Figure out what the real problem is and fix that instead. :)